app/src/main/java/io/gnosis/safe/ui/safe/settings/SafeSettingsViewModel.kt

```diff
@@ -26,16 +28,22 @@ class SafeSettingsViewModel @Inject constructor(
 
     fun removeSafe() {
         safeLaunch {
-            val safe = safeRepository.getActiveSafe()
-            safe?.let {
-                safeRepository.removeSafe(safe)
+            runCatching {
```

Comment:
Why is this in a `runCatching` block?

Reply:
no reason really. In case something goes wrong with removing